Stylesheets/common-css/main.css

```diff
@@ -0,0 +1,270 @@
+.page {
+    background-color: var(--st-white);
+}
+
+.page-container {
+    min-width: 83%;
+    max-width: 87%;
+    margin: auto;
+}
+
+nav {
+    padding: 2.5rem;
+    padding-left: 7.8rem;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    background-color: var(--st-primary-color);
+    border-bottom-right-radius: 15px;
+}
+
+hr {
+    margin-bottom: 2rem;
+    color: var(--st-background-grey);
+}
+
+.left-nav {
+    font-size: 3rem;
+    font-weight: 910;
```

Comment:
Always give font-weight in multiples of 100 or 50 because that is what usually is supported by lot of APIs